enhancements/worker-latency-profile/worker-latency-profile.md

```diff
@@ -0,0 +1,450 @@
+
+
+#### LowUpdateAndSlowReaction -> MediumUpdateAndAverageReaction
+| Kubelet Status Update Frequency | KCM Node Monitor Grade Period| KAS NotReady/Unreachable Toleration | Notes
+| -----------             | -----------                              | ----------- | ----------- |
+| 1m                 | 5m           |    60s       | LowUpdateAndSlowReaction (Initial state)
+| 1m                 | 2m           |     60s       | Kubelet will get only 2 chances to set the Status. **Higher chances of node incorrectly getting NotReady/Unreachable taint.**
+| 20s                 | 5m           |     60s       | No major impact on cluster health
+| 20s                 | 2m           |     60s       | MediumUpdateAndAverageReaction Profile (Final state)
+
+
+#### LowUpdateAndSlowReaction -> Default
+
+Because of the extremely high probability of node incorrectly getting NotReady/Unreachable taint, this transition is only allowed at day-0.
+
+| Kubelet Status Update Frequency | KCM Node Monitor Grade Period| KAS NotReady/Unreachable Toleration | Notes
+| -----------             | -----------                              | ----------- | -----------  |
+| 1m                 | 5m           |     60s       | LowUpdateAndSlowReaction Profile (Initial state)
+| 1m                 | 5m           |     300s       | No major impact on cluster health
+| 1m                 | 40s           |     60s       |  Kubelet will NOT get any chance to set the Status. **Node will incorrectly get NotReady/Unreachable taint.**
+| 1m                 | 40s           |     300s       | Kubelet will NOT get any chance to set the Status. **Node will incorrectly get NotReady/Unreachable taint.**
+| 10s                 | 40s           |    60s       |    No major impact on cluster health
+| 10s                 | 5m            |    300s       |  No major impact on cluster health
+| 10s                 | 5m           |     60s       | No major impact on cluster health
+| 10s                 | 40s           |    300s       |   Default Profile (Final state)
+
+For day-2 operation this transtion will result in **LowUpdateAndSlowReaction -> MediumUpdateAndAverageReaction -> Default** to to minimize the impact on the stability of the cluster.
+
+#### Support Procedures
+
+### Test Plan
+
+Testing should be thoroughly done at all levels, including unit, end-to-end, and integration.
+
+### Graduation Criteria
+
+#### Dev Preview -> Tech Preview
+
+`WorkerLatencyProfile` will be dev preview on its initial release. Internal and customer
+usage will be critical to gather information on bugs and enhancements to the
+underlying subsystem.
+
+Graduation requirements to Tech Preview are:
+
+* No regressions after applying the `WorkerLatencyProfile` in bringing up `Kubelet`, `Kube Controller Manager` and `Kube API Server`
+* No major imapct on the cluster stability while applying the `WorkerLatencyProfile`, especially nodes should not incorrectly get NotReady/Unreachable taint for prolonged period.
```

Comment:
except for the described disruptions as described in the previous section during rollout

Reply:
Updated, thanks.